framework/src/meshmodifiers/SideSetsAroundSubdomain.C

```diff
@@ -65,27 +61,13 @@ SideSetsAroundSubdomain::SideSetsAroundSubdomain(const InputParameters & paramet
 }
 
 void
-SideSetsAroundSubdomain::initialize()
-{
-  // Initialize the BlockRestrictable parent
-  initializeBlockRestrictable(_pars);
-}
-
-void
 SideSetsAroundSubdomain::modify()
 {
   // Reference the the libMesh::MeshBase
   MeshBase & mesh = _mesh_ptr->getMesh();
 
-  // Extract the 'first' block ID
-  SubdomainID block_id = *blockIDs().begin();
-
-  // Extract the SubdomainID
-  if (numBlocks() > 1)
-    mooseWarning("SideSetsAroundSubdomain only acts on a single subdomain, but multiple were "
-                 "provided: only the ",
-                 block_id,
-                 "' subdomain is being used.");
+  // Extract the block ID
+  SubdomainID block_id = (_mesh_ptr->getSubdomainIDs({getParam<SubdomainName>("block")}))[0];
```

Comment:
This PR looks ok except you may as well keep the warning message that is already there rather than just silently ignore all of the boundaries past the first one. I'm not entirely sure why `BlockRestrictable` doesn't work here. Maybe @aeslaughter can take a look but he's currently out and will be busy for the next several weeks. We don't have to hold this up if you need it.